### ISSUE TITLE
Add possibility for additional time interpolation options in surface obs pairing

### ISSUE THREAD

**Issue by rschwant** [NOAA-CSL/MELODIES-MONET#288]
Currently, surface observation pairing occurs when the obs and model time match exactly and then they are paired together. 

Having more options for time interpolation would be useful. For example, some sites provide data at 15 minute intervals and currently we are only using the hourly data. 

Certain models only provide instantaneous values every hour and determining exactly how to pair this with averaged hourly data from AirNow with more options would be useful. However, we have to be careful when implementing this to not disrupt the MDA8 or PM2.5 calculations.

It would also be useful to add a chart to ReadTheDocs to describe how each model provides the data and each obs provides the data so that users fully understand how the pairing works in the tool and can make the right decisions for how to link model and obs data together.

Let's think about doing some of the above at least the documentation in version 1 of surface and aircraft evaluation and some of this could be moved to version 2 as well.

**Comment by rschwant** [comment]
Zach has some ideas for an approximation we can add to better pair instantaneous model output with average obs data. We can add this in at some point as an option. Let's prioritize this as a quick fix for this and then we can do more advanced things in version 2.

**Comment by blychs** [comment]
Hi Becky, are you thinking of just starting with the drop? Would this issue be for after the tutorial then?

**Comment by rschwant** [comment]
Yes, this is just for after the tutorial. I think adding a quick fix for providing instantaneous output would be easy to add in. And adding a table to describe the observations with a reference for how the time is provided would also be pretty easy. And add an option to shift the hours if needed for certain datasets could also be possible. We have to figure out a way for users to use this correctly, but should be easy enough. The option to do linear interpolation, for surface obs we can save for version 2.

**Comment by rschwant** [comment]
For version 1, added an explanation of how the time interpolation works for the surface data to document current capability. The rest of this issue will be resolved in version 2.
